### PR TITLE
Remove columns and ones from slow2d transpose signatures

### PR DESCRIPTION
**Pull request by jbschlosser**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69044
* #68899
* __->__ #68898
* #68897

Differential Revision: [D32655873](https://our.internmc.facebook.com/intern/diff/D32655873)

### CODE REVIEW

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

File: aten/src/ATen/native/NaiveConvolutionTranspose2d.cpp [hunk 1]
```diff
@@ -281,20 +276,12 @@ void slow_conv_transpose2d_out_cpu_template(
...
   // Define a buffer of ones, for bias accumulation
-  // Note: this buffer can be shared with other modules, it only ever gets
-  // increased, and always contains ones.
-  if (ones.dim() != 2 ||
-      ones.size(0) * ones.size(1) < output_height * output_width) {
-    // Resize plane and fill with ones...
-    ones.resize_({output_height, output_width});
-    ones.fill_(1);
-  }
+  Tensor ones = bias.defined() ? at::ones({output_height, output_width}, input_.options()) : Tensor();
```
Comment (zou3519) [inline aten/src/ATen/native/NaiveConvolutionTranspose2d.cpp:284]: Hmm, is there a performance impact on this? I remember we benchmarked the slow conv finput changes, but did we benchmark this one as well?

Comment (zou3519) [reply aten/src/ATen/native/NaiveConvolutionTranspose2d.cpp:284]: It looks like we were creating column and ones on the fly previously anyways, so this shouldn't have a performance impact. Is this a correct assessment?

**Review by zou3519 — APPROVED** [review]
LGTM if my understanding is right

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32655873).

**Pull request closed without merging**